src/tiler/sql/user_progress.sql

```diff
@@ -0,0 +1,26 @@
+(SELECT
+  DISTINCT ON (block.id)
+  block.geom, block.id, turf.group_id,
+  CASE
+    WHEN (survey.user_id IS NOT DISTINCT FROM <%= user_id %>
```

Comment:
Surveys have to be associated with a user, but not every block will have a Survey.

The reason this column could be `NULL` is because our `LEFT OUTER JOIN` might not find any rows in the Survey table matching our Blockface.